packages/perseus/src/__testdata__/renderer.testdata.ts

```diff
@@ -4,7 +4,7 @@ import type {
     InputNumberWidget,
     PerseusRenderer,
 } from "../perseus-types";
-import type {RenderProps} from "../widgets/radio/widget";
+import type {RenderProps} from "../widgets/radio/radio";
```

Comment:
This rename came up while pairing on some of the coach report work. We ended up in widget.tsx, and found ourselves confused. The rename ensures that it's clearer that you're in the radio widget, and is more consistent with our other widgets.